_pages/testing.md

```diff
@@ -18,11 +18,11 @@ In the sandbox environment, our [Dashboard](https://dashboard.int.identitysandbo
 Anybody with an email address ending in .gov or .mil can create an account in the sandbox environment. If you are a government contractor, ask your agency partner to help you gain access. If you are with a government entity that is not a federal agency (a state or municipality) and do not have an email ending in .gov or .mil, please submit a support ticket through the [Partner Support Help Desk](https://zendesk.login.gov) to get access to the Dashboard. 
 
 1. Visit the dashboard at [https://dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov). Select the “Sign in” button in the upper-right corner to sign in or create a new account with the test Login.gov IdP in the sandbox environment hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov). Please note that this is separate from your Login.gov production account. 
-2. Once you are logged into your sandbox account, create a new team by selecting the “Continue” button under “Create your first team.” (If you have previously created a team you will select the “Create a new team” button.) Next you can add users to that team.
+2. Once you are logged into your sandbox account, create a new team by selecting the “Continue” button under “Create your first team". (If you have previously created a team you will select the “Create a new team” button.) Next you can add users to that team.
```

Comment:
Periods go inside of the quotation marks, at least according to MLA writing standards. And we don't do `".` anywhere else in the docs, but do `."` in two other places (1)[https://github.com/18F/identity-dev-docs/blob/34270bc871104a26825e5f134b78824afe65da17/_config.yml#L7](2)[https://github.com/18F/identity-dev-docs/blob/34270bc871104a26825e5f134b78824afe65da17/_pages/oidc.md?plain=1#L47].